document/Definitions.mdk

```diff
@@ -125,8 +125,12 @@ empty string
   ~ string that contains no characters, and so has a length of 0
 
 core format
-  ~ the set of SARIF elements that are intended to be useful in a variety of scenarios where static analysis results
+  ~ set of SARIF elements that are intended to be useful in a variety of scenarios where static analysis results
     are produced or consumed
 
 profile
-  ~ a set of SARIF elements that are intended to be useful in a specific static analysis usage scenario
\ No newline at end of file
+  ~ set of SARIF elements that are intended to be useful in a specific static analysis usage scenario
+
+policy
+  ~ set of requirements on a software system, or on the process used to produce it,
+    imposed by an authority such as project team management, a corporation, or a regulatory agency.
```

Comment:
nit. I think this would read better if the authorities were listed from most authoritative to least. i.e. 

"imposed by an authority such as a regulatory agency, a corporation, or project team management."